File: modul 1/tugas no 4 praktikum pemrogaraman 1/PRAK04.py
sepatuA = 400000
sepatuB = 350000
diskonA = 13
diskonB = 21

afterdiskonA = sepatuA - (sepatuA * diskonA / 100)
afterdiskonB = sepatuB - (sepatuB * diskonB / 100)

print("Harga sepatu A adalah", sepatuA)
print("Harga sepatu B adalah", sepatuB)
print(f"Sepatu A mendapat diskon {diskonA}% sehingga harganya menjadi {afterdiskonA:.0f} ")
print(f"Sepatu B mendapat diskon {diskonB}% sehingga harganya menjadi {afterdiskonB:.0f} ")

File: modul 1/tugas no 4 praktikum pemrogaraman 1/PRAK04.c
#include <stdio.h>

int main() {
    int sepatuA = 400000, sepatuB = 350000;
    int diskonA = 13, diskonB = 21;
    int AfterdiskonA, AfterdiskonB;

    AfterdiskonA = sepatuA - (sepatuA * diskonA / 100);
    AfterdiskonB = sepatuB - (sepatuB * diskonB / 100);

    printf("Harga sepatu A adalah %d\n", sepatuA);
    printf("Harga sepatu B adalah %d\n", sepatuB);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", diskonA, AfterdiskonA);
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", diskonB, AfterdiskonB);

    return 0;
}